apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx

```diff
@@ -49,6 +49,7 @@ const layoutSectionFlexStyle = {
       },
     },
   },
+  ...selectControlCssVars({ gridTemplateColumns: "auto 1fr" }),
```

Comment:
This is how we explicitely define the template columns for the select control, without ever running into specificity or namespace conflicts

Reply:
anybody who reads this can trace it back to the select component and know exactly who is using this